exporter/collector/metricsexporter.go

```diff
@@ -97,23 +123,82 @@ func (me *metricsExporter) pushMetrics(ctx context.Context, m pdata.Metrics) err
 			mes := ilm.Metrics()
 			for k := 0; k < mes.Len(); k++ {
 				metric := mes.At(k)
+				for _, md := range me.mapper.metricDescriptor(metric) {
+					me.mds <- md
+				}
 				timeSeries = append(timeSeries, me.mapper.metricToTimeSeries(monitoredResource, extraLabels, metric)...)
 			}
 		}
 	}
 
 	// TODO: self observability
+	// TODO: Figure out how to configure service time series calls.
+	if false {
```

Comment:
@aabmass @dashpole I added this as a reminder that I don't think we have an open bug to support service timeseries calls in this rework.   Would one of you mind confirming/opening approrpiately? I think David has the most context on what we need here.

Reply:
You can link to https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/225.

Reply:
You can probably just copy https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/pull/294/files#diff-9f6cddeabc26d57e3837fe66df90a6288f56e8a0e898cb6004d098da521583aeR656 and use it here.

Reply:
I assigned the bug to @jsuereth and pulled into the sprint

Reply:
sounds good, I'll work on that next.